akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/scaladsl/ActorTestKit.scala

```diff
@@ -185,6 +186,24 @@ final class ActorTestKit private[akka] (val name: String, val config: Config, se
    */
   def createTestProbe[M](name: String): TestProbe[M] = TestProbe(name)(system)
 
+  /**
+   * Verify serialization roundtrip.
+   * Throws exception from serializer if `obj` can't be serialized and deserialized.
+   *
+   * @param obj the object to verify
+   * @param assertEquality if `true` the deserialized  object is verified to be equal to `obj`,
+   *                       and if not an `AssertionError` is thrown
+   * @return the deserialized object
+   */
+  def verifySerialization[M](obj: M, assertEquality: Boolean): M = {
```

Comment:
I think a single param one with default `assertEquality` will be what (Scala) users need 99% of the time, so would be nice with that as a bonus.